src/arns.lua

```diff
@@ -615,7 +631,7 @@ function arns.getTokenCost(intendedAction)
 	if intent == "Buy-Record" then
 		-- stub the process id as it is not required for this intent
 		local processId = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
-		arns.assertValidBuyRecord(name, years, purchaseType, processId)
+		arns.assertValidBuyRecord(name, years, purchaseType, processId, false)
```

Comment:
this blocks getting a token cost for an unsafe name, we could use the allow-unsafe tag in this handler

Reply:
or might be better to have `true` in this read level API